### DISCUSSION TITLE
Filter and bulk action patterns

### DISCUSSION THREAD

**Discussion by johannaweintraub** [twilio-labs/paste#900] [General] [answered]
Are there any planned dates for filter and bulk action patterns?  Based on timeline, I'm trying to determine if certain designs can wait for the new pattern or if they should proceed with a one-off pattern.

**Comment by corinne-nagel** [comment, marked as answer]
Hi @johannaweintraub , you can check the Paste site here https://paste.twilio.design/patterns/ for the most up to date patterns work; while we don't have a defined date yet for filters, @serifluous is working on planning and prioritizing with  teams. In the meantime, we would love a contribution if you're seeing this has been causing a lot of confusion! Please refer to our contributions page here to get that going: https://paste.twilio.design/patterns/contribute

**Comment by johannaweintraub** [comment]
Great, thanks for the info!
